app/assets/javascripts/services/sse_factory.js

```diff
@@ -0,0 +1,20 @@
+samson.factory('SseFactory', function($location) {
+  'use strict';
+
+  var sse = {
+    connection: null,
+
+    init: function() {
+      this.connection = new EventSource($location.protocol() + '://' + $location.host() + ':' + $location.port() + '/sse');
```

Comment:
If the port was default, then it would still be '80', but I'll change it to the window.location.origin as that's cleaner.